Microsoft.NET.Build.Containers/ContentStore.cs

```diff
@@ -34,9 +34,11 @@ public static string PathForDescriptor(Descriptor descriptor)
         {
             "application/vnd.docker.image.rootfs.diff.tar.gzip"
             or "application/vnd.oci.image.layer.v1.tar+gzip"
+            or "application/vnd.docker.image.rootfs.foreign.diff.tar.gzip"
```

Comment:
This might not be required - I added it during a spot when I'd mixed up the RIDs accidentally.